tools/site_search.php

```diff
@@ -0,0 +1,343 @@
+        ],
+        "task" => [
+            "url" => "$code_url/tasks.php?task_id=%s",
+            "desc" => _("Jump to this task ID"),
+            "validator" => "_validate_integer",
+        ],
+    ];
+
+    if (get_url_for_forum()) {
+        $prefix_searches["forum"] = [
+            "url" => get_url_for_search("%s", false),
+            "desc" => _("Search for this in forum posts"),
+        ];
+    }
+
+    if (!empty($wiki_url)) {
+        $prefix_searches["wiki"] = [
+            "url" => "$wiki_url/Special:Search/Index.php?search=%s",
+            "desc" => _("Search for this in the wiki"),
+        ];
+    }
+
+    if ($flatten_aliases) {
+        foreach ($prefix_searches as $key => $data) {
+            if (isset($prefix_searches[$key]["alias"])) {
+                $prefix_searches[$prefix_searches[$key]["alias"]] = $data;
+                unset($prefix_searches[$key]["alias"]);
+            }
+        }
+    }
+
+    return $prefix_searches;
+}
+
+function get_jump_words(bool $flatten_aliases = false): array
+{
+    global $code_url, $wiki_url, $blog_url;
+
+    $jump_words = [
+        "activityhub" => [
+            "url" => "$code_url/activity_hub.php",
+            "desc" => _("Activity Hub"),
+            "alias" => "ah",
+        ],
+        "home" => [
+            "url" => "$code_url/default.php",
+            "desc" => _("Home Page"),
+        ],
+        "myprojects" => [
+            "url" => "$code_url/tools/proofers/my_projects.php",
+            "desc" => _("My Projects"),
+            "alias" => "myproj",
+        ],
+        "mysuggestions" => [
+            "url" => "$code_url/tools/proofers/my_suggestions.php",
+            "desc" => _("My Suggestions"),
+            "alias" => "mysugg",
+        ],
+        "preferences" => [
+            "url" => "$code_url/userprefs.php",
+            "desc" => _("Personal Preferences"),
+            "alias" => "prefs",
+        ],
+        "quizzes" => [
+            "url" => "$code_url/quiz/start.php",
+            "desc" => _("Interactive Quizzes and Tutorials"),
+            "alias" => "quiz",
+        ],
+        "statistics" => [
+            "url" => "$code_url/stats/stats_central.php",
+            "desc" => _("Statistics Central"),
+            "alias" => "stats",
+        ],
+        "tasks" => [
+            "url" => "$code_url/tasks.php",
+            "desc" => _("Task Center"),
+        ],
+    ];
+
+    if (get_url_for_forum()) {
+        $jump_words["forum"] = [
+            "url" => get_url_for_forum(),
+            "desc" => _("Forums"),
+        ];
+        $jump_words["inbox"] = [
+            "url" => get_url_for_inbox(),
+            "desc" => _("Inbox"),
+        ];
+    }
+
+    if (!empty($wiki_url)) {
+        $jump_words["wiki"] = [
+            "url" => $wiki_url,
+            "desc" => _("Wiki"),
+        ];
+    }
+
+    if (!empty($blog_url)) {
+        $jump_words["blog"] = [
+            "url" => $wiki_url,
```

Comment:
Is the blog supposed to point to the wiki?

Reply:
No, I just caught this myself!